thread.c

```diff
@@ -662,12 +656,31 @@ thread_start_func_2(rb_thread_t *th, VALUE *stack_start)
     // Ensure that we are not joinable.
     VM_ASSERT(UNDEF_P(th->value));
 
+    int rb_fiber_scheduler_set_invoked = 0, event_thread_end_hooked = 0;
```

Comment:
do not use `rb_` prefix for local variables.

Reply:
I don't really mind what we call it.